src/platform/w3dengine/client/w3ddisplay.cpp

```diff
@@ -60,80 +94,457 @@ W3DDisplay::W3DDisplay() :
 // 0x0073C453
 W3DDisplay::~W3DDisplay()
 {
-    // delete m_debugDisplay;
-    // for (auto &str : m_displayStrings) {
-    //    g_theDisplayStringManager->Free_Display_String(str);
-    //}
+    delete m_debugDisplay;
```

Comment:
nullptr check, yes msvc needs this

Reply:
Looked at the obj compiled by MSVC from this code and its got the nullptr check in there.